### PR TITLE
fix: Require a frozen lockfile when build

### LINKED ISSUE #65: Fix CI errors
https://github.com/node-casbin/sequelize-adapter/runs/8289139592?check_suite_focus=true

Cause by PR: https://github.com/node-casbin/sequelize-adapter/pull/64

image

### PR DESCRIPTION
**Pull request by js-lowes**
Fix: https://github.com/node-casbin/sequelize-adapter/issues/65

"--frozen-lockfile" will make it so that the yarn build process is reproducible and uses the versions of dependencies defined in the "yarn.lock" file.

From the [yarn docs](https://classic.yarnpkg.com/lang/en/docs/cli/install/):
> If you need reproducible dependencies, which is usually the case with the continuous integration systems, you should pass --frozen-lockfile flag.
> 
> ...
>
> yarn install --frozen-lockfile
>
> Don’t generate a yarn.lock lockfile and fail if an update is needed.

### CODE REVIEW

**Review by nodece — APPROVED** [review]

**Review by Shivansh-yadav13 — APPROVED** [review]

**Comment by js-lowes** [comment]
@nodece @Zxilly @Shivansh-yadav13 I can't merge, waiting on someone with write access to merge this

**Pull request merged**
